src/components/ChartContainer/D3Container.js

```diff
@@ -192,35 +276,34 @@ function D3Container({ dashboardState, dashboardActions, store }) {
         <Tab label="Measure by Line" className="d3-container__tab-button" />
       </Tabs>
       <TabPanel value={tabValue} index={1}>
-        <Grid container>
+        <Grid container justifyContent="space-evenly" direction="column">
```

Comment:
Candidate for CSS.

Reply:
That styling is duplicated too in the other tab panel.